dev-games/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Simulator Construction Toolkit"
+DESCRIPTION="Set of open-source libraries designed as building blocks \
+for quickly assembling 3D simulations, games, and visualization applications."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+
+PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
+	lib:libSimGearCored$secondaryArchSuffix = $libVersion
+	lib:libSimGearScened$secondaryArchSuffix = $libVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libosg$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	simgear$secondaryArchSuffix = $portVersion
+	devel:libSimGearCored$secondaryArchSuffix = $portVersion
+	devel:libSimGearScened$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	simgear$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_system$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libosg$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+defineDebugInfoPackage simgear$secondaryArchSuffix \
```

Comment:
Could you please add 3 empty lines?
```diff
 	"
+
 defineDebugInfoPackage simgear$secondaryArchSuffix \
 	"$libDir"/libSimGearCored.so.$libVersion \
 	"$libDir"/libSimGearScened.so.$libVersion \
+
+
 PATCH()
```
(I would have added only one before `PATCH()`, but some contributors prefer to have two, so let's go for 2 lines :)